src/extend-utils.ts

```diff
@@ -104,3 +105,44 @@ export function getWalletUtil(name: string, hre: HardhatRuntimeEnvironment) {
     wallet.accountPath = getAccountPath(wallet.accountPath, hre);
     return wallet;
 }
+
+export async function deployAccountFromABIUtil(
+    accountContract: string,
+    accountType: AccountTypes,
+    hre: HardhatRuntimeEnvironment
+): Promise<Account> {
+    let account: Account;
+    switch (accountType) {
+        case "OpenZeppelin":
+            account = await OpenZeppelinAccount.deployFromABI(accountContract, hre);
```

Comment:
Will not be needed if we decide to expose OpenZeppelinAccount class with its constructor.